packages/devtools_app/lib/src/theme.dart

```diff
@@ -242,6 +242,8 @@ extension DevToolsColorScheme on ColorScheme {
 
   Color get serviceExtensionButtonsFillSelected => devtoolsBlue[400];
 
+  Color get fileLineColColor => isLight ? Colors.black : Colors.white;
```

Comment:
is there a reason not to use defaultForeground instead? 
Using full white in the dark color scheme is brighter than other text so should be avoided unless there is a strong reason.

Reply:
My bad, was focused on creating own color for clarity. I will change it to use defaultForeground.